packages/composites/bob/src/index.ts

```diff
@@ -0,0 +1,6 @@
+import { makeConfig } from './config'
```

Comment:
CI check is failing because the Docker build fails from this line. Safe to remove this line and `makeConfig` from line 6.